### PR TITLE
Fixes #3289 - Add support for nested ul in browser details

### LINKED ISSUE #3289: Display GPU browser info as nested UL
Right now it will look like this:

`GPUs: [{'active': True, 'description': '', 'deviceID': '0x67ef', 'vendorID': '0x1002', 'driverVersion': ''}]`

Not the end of the world, but we could make this a nested UL to be more human friendly.

### PR DESCRIPTION
**Pull request by miketaylr**
(This will be used by the GPU data, if it exists)

I feel like Karl will have some opinions on this one, and it's cosmetic, so it can wait for him to come back.

### CODE REVIEW

File: webcompat/helpers.py [hunk 1]
```diff
@@ -85,10 +85,16 @@ def md5_checksum(file_path):
 
 
 def get_str_value(val):
```
Comment (karlcow) [inline webcompat/helpers.py:87]: This was created by #2395. And probably the name was not great at the start, but now it seems to have drifted a bit. We should probably change that to reflect the new semantics

And this is used by 
https://github.com/webcompat/webcompat.com/blob/9187af92432fc4046c9dcca39be5b6f009fabaea/webcompat/form.py#L296-L309

Comment (karlcow) [reply webcompat/helpers.py:87]: What I'm not sure is that are we trying to do twice the same thing at two different places. It looks very similar and confusing. 

Maybe our tests were not complete at the start.

`get_details` is being tested here

https://github.com/webcompat/webcompat.com/blob/9187af92432fc4046c9dcca39be5b6f009fabaea/tests/unit/test_form.py#L220-L227

Comment (miketaylr) [reply webcompat/helpers.py:87]: Yeah, it is a bit confusing. We're essentially going one more level deep so we handle a value that's more complex than a simple string.

**Comment by miketaylr** [comment]
OK, @karlcow. I refactored / renamed things a bit -- hopefully less confusing.

r? @karlcow

File: webcompat/helpers.py [hunk 2]
```diff
@@ -84,11 +84,42 @@ def md5_checksum(file_path):
         return m.hexdigest()
 
 
-def get_str_value(val):
-    """Map values from JSON to python."""
+def get_list_items(val_dict):
+    """Return list items (<li>) from the passed in list ([])."""
+    rv = ''.join(['<li>{k}: {v}</li>'.format(k=k, v=get_serialized_value(v))
+                 for k, v in list(val_dict.items())])
```
Comment (karlcow) [inline webcompat/helpers.py:90]: The code was inherited from probably the conversion from python 2 to python 3. IN this specific case `val_dict.items()` is a `<class 'dict_items'>` and is an iterable, which makes the `list()` wrapper unecessary

**Pull request merged**